packages/venia-ui/lib/components/CheckoutPage/checkoutPage.js

```diff
@@ -24,17 +25,15 @@ import ShippingMethod from './ShippingMethod';
 import ShippingInformation from './ShippingInformation';
 import OrderConfirmationPage from './OrderConfirmationPage';
 import ItemsReview from './ItemsReview';
+import payments from './PaymentInformation/paymentMethodCollection';
 import defaultClasses from './checkoutPage.css';
-import CheckoutPageOperations from './checkoutPage.gql.js';
 
 const errorIcon = <Icon src={AlertCircleIcon} size={20} />;
 
 const CheckoutPage = props => {
     const { classes: propClasses } = props;
     const { formatMessage } = useIntl();
-    const talonProps = useCheckoutPage({
-        ...CheckoutPageOperations
-    });
+    const talonProps = useCheckoutPage({});
```

Comment:
Shouldn't need to pass an object anymore.

Reply:
Changed to `const talonProps = useCheckoutPage();`